pkg/policyengine/connectivitypdp/connectivity_pdp.go

```diff
@@ -35,22 +36,23 @@ const (
 // WorkloadAttrs are the actual key-value attributes attached to any given workload.
 type WorkloadAttrs map[string]string
 
-// PDP is the main object to maintain a set of connectivity policies and decide
+// PDP is the main object to maintain a set of access policies and decide
 // whether a given connection is allowed or denied by these policies.
 type PDP struct {
 	privilegedPolicies policyTier
 	regularPolicies    policyTier
 }
 
-// policyTier holds a set of ConnectivityPolicies, split into deny policies and allow policies
+// policyTier holds a set of AccessPolicies, split into deny policies and allow policies
 // Within a tier, no two policies can have the same name, even if one is deny and the other is allow.
 type policyTier struct {
+	privileged    bool
```

Comment:
why needed? this is because you want to evaluate one tier before the other, correct?
Doesn't PDP already have two tiers named by their priorities?

Reply:
It's needed because the two tiers work slightly different when it comes to namespaces: the privileged tier doesn't care about policy namespace, while the unprivileged tier does.